Assets/Scripts/GameManager.cs

```diff
@@ -87,16 +85,30 @@ void Bounce(Character character)
     }
 
     void CheckRepTooLow()
+    {
+        if (tracker.Rep < -100)
```

Comment:
I think it'll be helpful to make this a constant so we can finagle with the numbers if required. This also applies to the major and minor problem numbers below.